shopping_mall/backend/scripts/migrate_json_to_faq_v2.py

```
@@ -0,0 +1,559 @@
+  ChromaDB       : {synced:3d}건 동기화
+
+다음 단계 — BM25 인덱스 재빌드:
+  uv run python ai/seed_rag.py --from-db
+
+이후부터는 어드민 UI(FaqPage)에서 FAQ를 관리하세요.
+""")
+
+    # 마이그레이션 완료 마커 저장
+    import datetime
+    marker.parent.mkdir(parents=True, exist_ok=True)
+    marker.write_text(f"completed at {datetime.datetime.now().isoformat()}\n")
```

Ruff (0.15.12)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)